src/components/ItemsList.js

```diff
@@ -118,7 +136,12 @@ const ItemsList = () => {
                         shoppingItemObject.lastPurchasedOn,
                       )}
                     />
-                    <label htmlFor={shoppingItemObject.shoppingListItemName}>
+                    <label
+                      htmlFor={shoppingItemObject.shoppingListItemName}
+                      aria-label={describedState(
+                        shoppingItemObject.daysLeftForNextPurchase,
+                      )}
```

Comment:
Similar to my comments above, I would recommend giving a more descriptive name. Something like `getShoppingItemDescription` or `getItemDescription` would go a long way in making the code self-explanatory.

Reply:
Nice suggestion, the variable name has now been updated